### PR TITLE
Text snippets for sudo and management accounts retrieval 

### PR DESCRIPTION
**Pull request by augustocollado**
### Description

This PR creates two snippets, one for retrieving the sudo account and another one for retrieving the management account.

In a separate PR, I'll be addressing the removal of the images that were added to the scope of every article to cover these now unified topics, and reorder each article's images to start from "image-name-1.webp".
I'll also reorder the images in the drive after this is accepted :)

### Checklist

- [ ] I have added a label to this PR 🏷️
- [ ] I have run my changes through Grammarly
- [ ] If this page requires a disclaimer, I have added one
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects

### CODE REVIEW

**Review by eshaben — COMMENTED** [review]
Can you please crop these images a bit more so the background doesn't show up like a border?

Screenshot 2024-06-17 at 5 38 14 PM

File: builders/manage/dapp/manage-tokens.md
Comment (albertov19) [reply builders/manage/dapp/manage-tokens.md:34]: Unfortunately, for the Appchain to show in the dashboard you need the Management account connected. And then you need the Sudo account to be able to do the actual manage token actions

Comment (albertov19) [reply builders/manage/dapp/manage-tokens.md:34]: @augustocollado maybe we need to clarify this

**Review by albertov19 — CHANGES_REQUESTED** [review]
Found another that can be set as a snippet I believe

/builders/manage/dapp/manage-tokens/#checking-prerequisites
http://localhost:8000/builders/manage/dapp/xcm-channels/#checking-prerequisites


Here -> /builders/manage/dapp/services-payment/#checking-prerequisites

W also have something that is similar to other pages. For future reference, that is why ti is important to do it when writing the article and not later, because then it is hard to remember what can go in a snippet etc.

image

File: .snippets/text/builders/manage/locate-sudo-account.md
Comment (eshaben) [inline .snippets/text/builders/manage/locate-sudo-account.md:1]: Sometimes we use sudo and sometimes we use appchain governor. I feel like using "appchain governor" was an afterthought and now there's a slight disconnect and sometimes we use them interchangeably but never state that they can be one in the same

File: builders/manage/dapp/manage-tokens.md
Comment (augustocollado) [reply builders/manage/dapp/manage-tokens.md:34]: (Adding to what Alberto said, both accounts might be different, registration is always a substrate type and sudo could be evm)

File: .snippets/text/builders/manage/locate-sudo-account.md
Comment (augustocollado) [reply .snippets/text/builders/manage/locate-sudo-account.md:1]: In my mind, when referring to the account, I think of sudo. When I think of privileged actions, It's the appchain governor that executes them.
Maybe we should start using something like "governor's account" instead of sudo?

Comment (albertov19) [reply .snippets/text/builders/manage/locate-sudo-account.md:1]: I don't like governor's because it is not that self-explanatory. For now, I would stick to SUDO. The main problem being that SUDO is not long-lived because in the future there might be decentralized chains? 

Unsure

**Review by albertov19 — APPROVED** [review]
LGTM - Wait for Erin's approval plz :)

**Review by eshaben — COMMENTED** [review]
LGTM!

**Pull request merged**
